src/indicator/algorithm/calculator/stochasticOscillator.js

```diff
@@ -56,13 +56,8 @@ export default function() {
         return stochastic;
     };
 
-    rebind(stochastic, kWindow, {
-        kWindowSize: 'windowSize'
-    });
-
-    rebind(stochastic, dWindow, {
-        dWindowSize: 'windowSize'
-    });
+    rebindAll(stochastic, kWindow, map({'windowSize': 'kWindowSize'}));
+    rebindAll(stochastic, dWindow, map({'windowSize': 'dWindowSize'}));
```

Comment:
That's correct, `map` has an implicit `include` and from what I've seen in d3fc, that's what's required. If I remove the implicit `include` I reckon there'll be a lot of duplication of property names. Do you think more of a warning in the docs would help or maybe a rename to `onlyMap`/`includeMap` or something else?